google-cloud-language/.yardopts

```diff
@@ -1,6 +1,5 @@
 --no-private
 --title=Google Cloud Language
---exclude lib/google/cloud/language/v1
 --exclude lib/google/cloud/language/v1.rb
```

Comment:
Sorry, misunderstood your question. The `lib/google/cloud/<package>/<version>.rb` always needs to be excluded so that the docs from types in the `doc` directory are not overwritten.